build/schema.graphql

```diff
@@ -31,3 +31,17 @@ type Unpause @entity(immutable: true) {
   blockTimestamp: BigInt!
   transactionHash: Bytes!
 }
+
+type Request @entity(immutable: true) {
```

Comment:
We also need the payment made to the mech.

Reply:
The Request event does not include any additional information. Is there a way to fetch the information using the mech contract and request ID?

Reply:
Please check with Ardian

Reply:
The request event doesnt specify the fee. The destination address (fee receiver i assume) is always the mech emitting the event.

Reply:
Pricing is always 0.01xdai for the time being. It's not reflected in any event atm. In the future when we have a dynamic pricing mech, the events will likely include that.

Reply:
ok I suggest open an issue not to forget about it.